packages/astropy/test_astropy.py

```diff
@@ -0,0 +1,48 @@
+import pytest
+from pyodide_test_runner import run_in_pyodide
+
+
+@pytest.mark.parametrize(
+    "package,specific_test",
+    [
+        ("constants", "test_"),
+        ("units", "test_physical.py"),
+        ("units", "test_quantity.py"),
+        ("units", "test_quantity_ufuncs.py"),
+        ("units", "test_quantity_non_ufuncs.py"),
+        ("units", "test_units.py"),
+        ("coordinates", "test_angles.py"),
+        ("coordinates", "test_angular_separation.py"),
+        ("coordinates", "test_arrays.py"),
+        ("coordinates", "test_celestial_transformations.py"),
+        ("coordinates", "test_erfa_astrom.py"),
+        ("coordinates", "test_frames_with_velocity.py"),
+        ("coordinates", "test_icrs_observed_transformations.py"),
+        ("coordinates", "test_matching.py"),
+        ("coordinates", "test_representation.py and not test_unitphysics"),
+        ("coordinates", "test_sky_coord.py"),
+        ("coordinates", "test_sky_coord_velocities.py"),
+        ("coordinates", "test_transformations.py"),
+        ("cosmology", "test_base.py"),
+        ("cosmology", "test_lambdacdm.py"),
+        ("cosmology", "test_units.py"),
+    ],
+)
+@pytest.mark.asyncio
```

Comment:
```suggestion
```

I think I was mistaken. This marker is not needed.

Reply:
Yes, it seems like this isn't necessary, I guess because of the way `run_in_pyodide` is written.